src/typeahead/typeahead.ts

```diff
@@ -284,13 +285,20 @@ export class NgbTypeahead implements ControlValueAccessor,
       this._windowRef.instance.selectEvent.subscribe((result: any) => this._selectResultClosePopup(result));
       this._windowRef.instance.activeChangeEvent.subscribe((activeId: string) => this.activeDescendant = activeId);
 
+      const nativeElement = this._windowRef.location.nativeElement;
```

Comment:
Can you please rename `nativeElement` → `windowElement` and add a 1-2 line comment on why have this 'mousedown'. I don't think anybody will remember this in a year :)